server/routes/workers.js

```diff
@@ -43,7 +43,7 @@ router.post('/', async function (req, res, next) {
 router.delete('/', async function (req, res, next) {
   try {
     const workerId = req.body.id;
```

Comment:
i think this should be `const workerId = req.body._id`?
since we removed the `id` and the existing worker (we have added) should have an `_id` field

Reply:
Will fix after this weekend.